translate/cloud-client/src/test/java/com/example/translate/HybridGlossariesIT.java

```diff
@@ -0,0 +1,154 @@
+/*
+ * Copyright 2019 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.example.translate;
+
+import static com.google.common.truth.Truth.assertThat;
+
+import com.google.cloud.translate.v3beta1.DeleteGlossaryRequest;
+import com.google.cloud.translate.v3beta1.Glossary;
+import com.google.cloud.translate.v3beta1.ListGlossariesRequest;
+import com.google.cloud.translate.v3beta1.ListGlossariesResponse;
+import com.google.cloud.translate.v3beta1.LocationName;
+import com.google.cloud.translate.v3beta1.TranslationServiceClient;
+import com.google.cloud.translate.v3beta1.TranslationServiceClient.ListGlossariesPagedResponse;
+
+import java.io.ByteArrayOutputStream;
+import java.io.File;
+import java.io.PrintStream;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.UUID;
+import java.util.concurrent.TimeUnit;
+
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+
+/**
+ * Tests for HybridGlossaries sample.
+ */
+@RunWith(JUnit4.class)
+@SuppressWarnings("checkstyle:abbreviationaswordinname")
+public class HybridGlossariesIT {
+
+  private static String PROJECT_ID = System.getenv("GOOGLE_CLOUD_PROJECT");
+  private static String OUTPUT_FILE = "output.mp3";
+  private static String INPUT_FILE = "resources/test.jpeg";
+  private static String TEXT_FILE = "resources/test.txt";
+  private static List<String> LANGUAGES;
+  private static String SRC_LANG = "fr";
+  private static String TGT_LANG = "en";
+  private static String GLOSSARY_NAME;
+  private static String GLOSSARY_URI =
+      "gs://cloud-samples-data/translation/bistro_glossary.csv";
+
+  private ByteArrayOutputStream bout;
+  private PrintStream out;
+  private File outputFile;
+
+  @Before
+  public void setUp() {
+    GLOSSARY_NAME = String.format("bistro-glossary-%s", UUID.randomUUID());
+    LANGUAGES = new ArrayList<>();
+    LANGUAGES.add(SRC_LANG);
+    LANGUAGES.add(TGT_LANG);
+    // collect project ID from environment
+    bout = new ByteArrayOutputStream();
+    out = new PrintStream(bout);
+    System.setOut(out);
+  }
+
+  @After
```

Comment:
Should this happen after each test? or should it be AfterClass?

Reply:
Is this still a problem?

Reply:
Since 2 tests create glossaries. 
I can either in `setup` create UUID for the glossary names and run it after the class 
or run it each time.

I'll move the GLOSSARY_DISPLAY_NAME back to the setup

Reply:
I'm okay with either, I just wanted to make sure it was intended.